mathesar/urls.py

```diff
@@ -7,16 +7,20 @@
 router = routers.DefaultRouter()
 router.register(r'tables', api.TableViewSet, basename='table')
 router.register(r'schemas', api.SchemaViewSet, basename='schema')
-router.register(r'database_keys', api.DatabaseKeyViewSet, basename='database_keys')
+router.register(r'database_keys', api.DatabaseKeyViewSet, basename='database-key')
 router.register(r'data_files', api.DataFileViewSet)
 
 records_router = routers.NestedSimpleRouter(router, r'tables', lookup='table')
-records_router.register(r'records', api.RecordViewSet, basename='table-records')
+records_router.register(r'records', api.RecordViewSet, basename='table-record')
+
+columns_router = routers.NestedSimpleRouter(router, r'tables', lookup='table')
```

Comment:
Can't you rename the `records_router` to `table_router` or something like that and register both the records and columns to the same router rather than making a new one?